### ISSUE TITLE
IE 3.14.0 Returning 405's on Execute

### ISSUE THREAD

**Issue by ryanthemanuel** [webdriverio/webdriverio#2983]
## The problem

While running WDIO tests against a 3.14.0 IE selenium grid, I am receiving the following error:

```
details: undefined,
  message: 'java.io.IOException: Server returned HTTP response code: 405 for URL: http://localhost:8742/session/5284248d-87b7-48f9-81d6-ffb6795d3722/execute',
  type: 'RuntimeError',
  seleniumStack: 
   { type: 'UnknownError',
     message: 'An unknown server-side error occurred while processing the command.',
     orgStatusMessage: 'java.io.IOException: Server returned HTTP response code: 405 for URL: http://localhost:8742/session/5284248d-87b7-48f9-81d6-ffb6795d3722/execute' } }
{ Error: java.io.IOException: Server returned HTTP response code: 405 for URL: http://localhost:8742/session/5284248d-87b7-48f9-81d6-ffb6795d3722/execute
    at new RuntimeError (/Users/rm012685/terra/orion-toolkit-js/node_modules/webdriverio/build/lib/utils/ErrorHandler.js:143:12)
    at Request._callback (/Users/rm012685/terra/orion-toolkit-js/node_modules/webdriverio/build/lib/utils/RequestHandler.js:316:39)
    at Request.self.callback (/Users/rm012685/terra/orion-toolkit-js/node_modules/request/request.js:185:22)
    at emitTwo (events.js:126:13)
    at Request.emit (events.js:214:7)
    at Request.<anonymous> (/Users/rm012685/terra/orion-toolkit-js/node_modules/request/request.js:1157:10)
    at emitOne (events.js:116:13)
    at Request.emit (events.js:211:7)
    at IncomingMessage.<anonymous> (/Users/rm012685/terra/orion-toolkit-js/node_modules/request/request.js:1079:12)
    at Object.onceWrapper (events.js:313:30)
    at emitNone (events.js:111:20)
    at IncomingMessage.emit (events.js:208:7)
    at endReadableNT (_stream_readable.js:1056:12)
    at _combinedTickCallback (internal/process/next_tick.js:138:11)
    at process._tickDomainCallback (internal/process/next_tick.js:218:9)
```

## Environment

* WebdriverIO version: 4.13.2
* Node.js version: 8.9.2
* Standalone mode or WDIO Testrunner: WDIO Testrunner
* if wdio testrunner, running synchronous or asynchronous tests: synchronous

**Comment by emilyrohrbough** [comment]
I have experienced this issue with standalone mode as well.

**Comment by christian-bromann** [comment]
Fixed by #2990 thanks @emilyrohrbough
